spring-kafka/src/main/java/org/springframework/kafka/streams/KafkaStreamsInteractiveQueryService.java

```diff
@@ -75,20 +81,79 @@ public void setRetryTemplate(RetryTemplate retryTemplate) {
 	 * @return queryable store.
 	 */
 	public <T> T retrieveQueryableStore(String storeName, QueryableStoreType<T> storeType) {
+		primeKafkaStreamsObject();
+		StoreQueryParameters<T> storeQueryParams = StoreQueryParameters.fromNameAndType(storeName, storeType);
+
+		return this.retryTemplate.execute(context -> {
+			try {
+				return this.kafkaStreams.store(storeQueryParams);
+			}
+			catch (Exception e) {
+				throw new IllegalStateException("Error retrieving state store: " + storeName, e);
+			}
+		});
+	}
+
+	private void primeKafkaStreamsObject() {
```

Comment:
`setupKafkaStreams()` or `populate`?
I don't see a reason in `Object` suffix at all.
And never saw `prime` as a verb 😃

Reply:
Prime can be a verb, but I get your point:)